Library/ViewModels/RewardAddOnSelectionViewModel.swift

```diff
@@ -85,34 +104,80 @@ public final class RewardAddOnSelectionViewModel: RewardAddOnSelectionViewModelT
       .takePairWhen(latestSelectedQuantities)
       .map(rewardsData)
 
-    self.shippingLocationViewIsHidden = reward.map(\.shipping.enabled)
+    self.shippingLocationViewIsHidden = baseReward.map(\.shipping.enabled)
       .negate()
 
-    let totalSelectedQuantity = latestSelectedQuantities.map { quantities in
-      quantities.reduce(0) { accum, keyValue in
-        let (_, value) = keyValue
-        return accum + value
-      }
+    let totalSelectedAddOnsQuantity = Signal.combineLatest(
+      latestSelectedQuantities,
+      baseReward
```

Comment:
Is there a need to have the base reward here if it's filtered out after?

Reply:
Yeah, we need it to know its ID in order to filter it out.